scripts/setup.sh

```diff
@@ -188,15 +188,17 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Install xvfb, used in frontend, e2e tests and performance tests.
+sudo apt-get install xvfb
```

Comment:
Will this work on other Linux distros? What if a Linux distro doesn't have the ``apt-get`` package manager?